src/components/ResultListItem.tsx

```diff
@@ -123,11 +123,14 @@ const ResultListItem: FC<ResultListItemProps> = ({ resource }) => {
     <StyledListItem data-testid={`list-item-resources-${resource.identifier}`}>
       <StyledFirstColumn>
         <StyledThumbnailWrapper>
-          <Link data-testid={`thumbnail-link-to-resource-${resource.identifier}`} href={generateURL(resource)}>
+          <Link
```

Comment:
Får fortsatt erroren, men nå skal ikke skjermleser brukere få lest opp lenka to ganger.